### ISSUE TITLE
support Android sdk build-tools apksigner v35

### ISSUE THREAD

**Issue by sisong** [sisong/ApkDiffPatch#96]
apksigner v34 compatible ok, but v35  patcher failed.

**Comment by sisong** [comment]
now, apksigner v35 will  automatically realigned apk's uncompressed files ):    
and the alignment is different from zipalign!   
for all uncompressed files, apksigner v35 will adding  align tag+ align bytes+align multiple (It can also be handled compatible),
for all uncompressed .so files,  apksigner v35 will adding  align tag datas twice (no pattern  of location, and it can't be compatible):

![Image_1722907408498.png](https://github.com/user-attachments/assets/17992736-5ad0-4ba5-a778-1955736a63c8)

![Image_1722907411571.png](https://github.com/user-attachments/assets/86dfaa21-61a9-49de-81a1-981b974f5f41)

**Comment by jay-dh** [comment]
> apksigner v34 compatible ok, but v35 patcher failed.

If my APK is adapted to 16k page size, will apksigner v35 work ok?

**Comment by sisong** [comment]
No; apksigner v35 will re-enforce 16k page size alignment regardless of whether 16k page size alignment has been adapted.
